src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Customer.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource
+        class="Sylius\Component\Core\Model\Customer"
+        shortName="Customer"
+    >
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>customer:read</attribute>
+            </attribute>
+        </attribute>
+        <itemOperations>
+            <itemOperation name="get">
```

Comment:
As I understand, it's required to have `Customer`'s (and `Product`s) IRI embedded within a `ProductReview`?

Reply:
yes, without this basic configuration APi platform serialize all related entity :( I hope this is answer for your question :D

Reply:
yes, without this basic configuration APi platform serialize all related entity :( I hope this is answer for your question :D